extensions/chathistory-3.3.md

```diff
@@ -8,6 +8,13 @@ copyrights:
     period: "2017"
     email: "evan@muffinmedic.net"
 ---
+## Notes for implementing work-in-progress version
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the unprefixed `CHATHISTORY` command. Instead, implementations SHOULD use the `draft/CHATHISTORY` command to be interoperable with other software implementing a compatible work-in-progress version.
```

Comment:
The extension itself is lowercase. Only thee command itself is capitalized for consistency (CHATHISTORY, PRIVMSG, NOTICE, etc...).